projects/addon-doc/src/components/main/main.template.html

```diff
@@ -17,6 +17,7 @@
                 (ngModelChange)="onMode($event)"
             ></tui-toggle>
         </label>
+        <tui-language-switcher></tui-language-switcher>
```

Comment:
Now it looks like the component switches the language of the entire site (including navigation etc) but it doesn't. Maybe move the switcher closer to components?

Reply:
I'm not sure 
but yes, part of the functionality is translated ))

<img width="884" alt="image" src="https://user-images.githubusercontent.com/12021443/182315608-b9456307-69f1-4676-9bae-0a950df5996a.png">

Reply:
<img width="658" alt="image" src="https://user-images.githubusercontent.com/12021443/182315829-241776b8-4852-44bd-89d9-f7e41873849f.png">